### ISSUE TITLE
Incompatible with scikit-learn version 1.1

### ISSUE THREAD

**Issue by mononitogoswami** [TheDatumOrg/VUS#2]
Hi, Thank you for the excellent work! I am facing a scikit-learn version error. Would it be possible that VUS worked with the latest version of scikit-learn?

**Comment by boniolp** [comment]
Hi, some of the algorithms in our benchmark require an older version of scikit-learn. Though, I will work now to update all algorithms to work with the latest version of scikit-learn.

**Comment by fede-bello** [comment]
Hi, is this going to happen? It's still working only with a really old version of scikit-learn. 
Loved the paper btw

**Comment by boniolp** [comment]
Hi @fede-bello,

Thanks for your interest (and for the reminder).
We pushed a more recent version of VUS (working with sklearn above 1.1) in the TSB-UAD repository (https://github.com/TheDatumOrg/TSB-UAD). 

You can install the package with "pip install tsb-uad" and use vus as follows:

```(python)
from TSB_UAD.vus.metrics import get_metrics

results = get_metrics(score, label, metric="all", slidingWindow=slidingWindow)
```

The latter returns a dictionary including VUS-ROC and VUS-PR (you can change the parameter metric to return only VUS-based measures). Below is an example of the returned dictionary:

```
AUC_ROC : 0.9216216369841076
AUC_PR : 0.6608577550833885
Precision : 0.7342093339374717
Recall : 0.4010891089108911
F : 0.5187770129662238
Precision_at_k : 0.4010891089108911
Rprecision : 0.7486112853253205
Rrecall : 0.3097733542316151
RF : 0.438214653167952
R_AUC_ROC : 0.989123018780308
R_AUC_PR : 0.9435238401582703
VUS_ROC : 0.9734357459251715
VUS_PR : 0.8858037295594041
Affiliation_Precision : 0.9630674176380548
Affiliation_Recall : 0.9809813654809071
```

I will push soon the same version on this repo as well.

Best,

Paul

**Comment by fede-bello** [comment]
That's awesome, thanks!

**Comment by boniolp** [comment]
@qhliu26 just updated the repo to support sklearn 1.1 or above.
